src/pages/profiles/modification/profile-modification-form.tsx

```diff
@@ -37,6 +38,22 @@ const ProfileModificationForm: FunctionComponent = () => {
                     parameterFormId={LF_PARAM_ID}
                 />
             </Grid>
+            <Grid item xs={12}>
+                <h3>
+                    <FormattedMessage
+                        id={'profiles.form.modification.userQuotas'}
+                    />
+                </h3>
+            </Grid>
+            <Grid item xs={12}>
+                <Grid item xs={4}>
```

Comment:
When french language is selected, we don't see the label inside the input field in its entirety ...
xs={5} solve this problem ...